UnityProject/Assets/Scripts/Objects/Engineering/APCPoweredDevice.cs

```diff
@@ -60,14 +62,8 @@ public class APCPoweredDevice : NetworkBehaviour, IServerDespawn, IEmpAble, IMul
 		[FormerlySerializedAs("resistance")]
 		private float InitialResistance = 99999999;
 
-
-		private float resistance = 99999999;
-
-
-		public float Resistance {
```

Comment:
I think this was meant to trigger resistance change?

Reply:
Can't find any function in the class for a resistance change?